docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -30,7 +30,11 @@ def _filter(obj, n):
                 "set_numeric_ops",
                 "size",
                 "sometrue",
-            ]  # not in blacklist
+                "loads",
+                "mafromtxt",
+                "matmul",
+                "ndfromtxt",
+            ]  # not in blocklist
```

Comment:
I believe the comment should be `# not in blacklist`.

Reply:
@magnatelee This edit was intentional. There has been a concerted effort across OSS in the last several years to move towards more inclusive terminology (refs: [[1]](https://developers.google.com/style/inclusive-documentation), [[2]](https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/), [[3]](https://tools.ietf.org/id/draft-knodel-terminology-00.html This has included other Nvidia OSS projects (ref: https://github.com/NVIDIA/spark-rapids/issues/363) as well as Python itself (https://bugs.python.org/issue41521).

I would suggest keeping this edit.

Reply:
Your comment actually made me realized of such an effort that I completely forgot about. I agree we should keep that edit.